ParselyTrackerTests/SamplerTests.swift

```diff
@@ -126,4 +126,31 @@ class SamplerTests: ParselyTestCase {
         XCTAssertNotNil(samplerUnderTest!.samplerTimer,
                         "After a call to Sampler.resume() without sampling having started, Sampler.samplerTimer should be nil")
     }
+    
+    func testPauseStopsCounting() {
+        let assertionTimeout:TimeInterval = TimeInterval(3)
+        let acceptableDifference:TimeInterval = TimeInterval(0.2)
+        
+        samplerUnderTest!.trackKey(key: "sampler-test", contentDuration: nil, eventArgs: [:])
```

Comment:
Another implicitly unwrapped optional. Maybe a future iteration on the SDK could have a look at these.